src/debugAdapter/goDebug.ts

```diff
@@ -2276,4 +2289,21 @@ async function removeFile(filePath: string): Promise<void> {
 	}
 }
 
+function killProcessTree(p: ChildProcess): Promise<void> {
```

Comment:
There are already similar looking functions under util.ts:
https://github.com/golang/vscode-go/blob/9f328b7cbe80a1be66f37faf71f31678fa1d9d62/src/util.ts#L857
You should either reuse them or add a new one there.

Reply:
goDebug is a separate node.js process outside of vscode, so can't depend on util.ts which requires the `vscode` module.

Probably, the killTree in util.ts is also problematic and needs fixing. I don't want to do it in this change.